pkg/operator/hostetcdendpointcontroller/members.go

```diff
@@ -86,12 +86,12 @@ func (h *healthyEtcdMemberGetter) EtcdList(bucket string) ([]ceoapi.Member, erro
 	if err != nil {
 		return nil, err
 	}
+	// populate current etcd members as observed.
+	members := []ceoapi.Member{}
 	if !exists {
-		return nil, fmt.Errorf("etcd cluster members not observed")
+		return members, nil
```

Comment:
If there is no error, but exists is `false` you are returning an empty structure, right? But the comment is misleading.

Reply:
yeap, done, thanks!